### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by neilish3re**
Converted AB4 to Concierge v1.1

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -18,13 +18,16 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
...
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+Concierge is created for *hotel managers and receptionists* of small hotels looking for a simple application to *assist
+the user in hotel management*. More importantly, Concierge is *optimized for those who prefer to work with a Command Line Interface*
+(CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Concierge can get your
+hotel management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
 
 == Quick Start
 
 .  Ensure you have Java version `9` or later installed in your Computer.
 .  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
```
Comment (JiaqingTan) [inline docs/UserGuide.adoc]: Might wanna change "addressbook.jar" to "concierge.jar"!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -33,9 +36,9 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`list`* : lists all contacts
+* *`list`* : lists all rooms types and cost of rooms
 * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
```
Comment (JiaqingTan) [inline docs/UserGuide.adoc]: I think can remove this line!

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -56,47 +59,133 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]...`
 
 [TIP]
-A person can have any number of tags (including 0)
+A guest can have any number of tags (including 0)
 
 Examples:
 
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
```
Comment (JiaqingTan) [inline docs/UserGuide.adoc]: Can update the above 2 to show the "add guest" format!

Comment (teowz46) [reply docs/UserGuide.adoc]: wait do our guests even have tags

Comment (adamwth) [reply docs/UserGuide.adoc]: we can include it since the feature's already been implemented
(e.g. tag people with disabilities)

**Pull request merged**